tsql/TSqlParser.g4

```diff
@@ -2779,7 +2805,7 @@ expression
     | bracket_expression
     | unary_operator_expression
     | expression op=('*' | '/' | '%') expression
-    | expression op=('+' | '-' | '&' | '^' | '|') expression
+    | expression op=('+' | '-' | '&' | '^' | '|' | DOUBLE_BAR) expression
```

Comment:
Why not an ordinary introduced string literal `||`?

Reply:
I prefer terminals to literals for operators. But if you insist…

Reply:
There are a mix of ordinary literals and identifiers. Please use the certain ones.

Reply:
Tried that, got an error: ```cannot create implicit token for string literal in non-combined grammar: '||'```. If you know, how to do it I’d be happy to learn.

Reply:
Strange. Such error occurs if the token is not defined explicitly in Lexer. Did you remove introduced `DOUBLE_BAR` token?

Reply:
Yes, I did indeed. Committing now.